### ISSUE TITLE
Error - No datepickerAdapter provided for picker. nebular nb-date-timepicker not working properly with default configuration

### ISSUE THREAD

**Issue by Tavethiya** [akveo/nebular#2603]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [X] bug report
* [ ] feature request

###  nebular nb-date-timepicker not working properly with default configuration: 

**Current behavior:**
nb-date-timepicker not open on textbox 

**Expected behavior:**
nb-date-timepicker should work fine same as nb-date-picker

**Steps to reproduce:**
Check bellow link provided open console and click on textbox error will occur as No Adapter provided for picker 
I don't know how to resolve this error no doc provided on online documents 

**Related code:**

A sample application

StackBlitz [StackBlitz Seed Project](https://stackblitz.com/edit/nebular-date-time-picker?file=src/app/app.component.ts)

```
<input nbInput placeholder="Pick Date" [nbDatepicker]="dateTimePicker11">
         <nb-date-timepicker withSeconds  #dateTimePicker11>
</nb-date-timepicker>
```

### Other information:

**npm, node, OS, Browser**
```
node -v
v12.18.3

npm -v  
6.14.6

```

**Angular, Nebular**
```
  "dependencies": {
    "@akveo/ng2-completer": "^9.0.1",
    "@angular/animations": "^10.0.10",
    "@angular/cdk": "10.1.1",
    "@angular/common": "^10.0.10",
    "@angular/compiler": "^10.0.10",
    "@angular/core": "^10.0.10",
    "@angular/forms": "^10.0.10",
    "@angular/material": "^10.1.3",
    "@angular/platform-browser": "^10.0.10",
    "@angular/platform-browser-dynamic": "^10.0.10",
    "@angular/router": "^10.0.10",
    "@asymmetrik/ngx-leaflet": "3.0.1",
    "@nebular/auth": "6.0.0",
    "@nebular/date-fns": "^6.2.1",
    "@nebular/eva-icons": "6.0.0",
    "@nebular/moment": "^6.2.1",
    "@nebular/security": "6.0.0",
    "@nebular/theme": "6.0.0",
    "@ngx-translate/core": "^13.0.0",
    "@ngx-translate/http-loader": "^6.0.0",
    "@swimlane/ngx-charts": "^14.0.0",
    "angular2-chartjs": "0.4.1",
    "bootstrap": "4.3.1",
    "chart.js": "2.7.1",
    "ckeditor": "4.7.3",
    "classlist.js": "1.1.20150312",
    "core-js": "2.5.1",
    "date-fns": "^2.16.1",
    "echarts": "^4.0.2",
    "eva-icons": "^1.1.3",
    "intl": "1.2.5",
    "ionicons": "2.0.1",
    "leaflet": "1.2.0",
    "nebular-icons": "1.1.0",
    "ng2-ckeditor": "^1.2.9",
    "ng2-smart-table": "^1.6.0",
    "ngx-echarts": "^4.2.2",
    "node-sass": "^4.12.0",
    "normalize.css": "6.0.0",
    "pace-js": "1.0.2",
    "primeflex": "^2.0.0",
    "primeicons": "^4.0.0",
    "primeng": "^10.0.2",
    "quill": "^1.3.7",
    "roboto-fontface": "0.8.0",
    "rxjs": "^6.6.2",
    "rxjs-compat": "6.3.0",
    "socicon": "3.0.5",
    "style-loader": "^1.1.3",
    "tinymce": "4.5.7",
    "tslib": "^2.0.0",
    "typeface-exo": "0.0.22",
    "web-animations-js": "^2.3.2",
    "zone.js": "~0.10.2"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.1000.6",
    "@angular/cli": "^10.0.6",
    "@angular/compiler-cli": "^10.0.10",
    "@angular/language-service": "10.0.10",
    "@compodoc/compodoc": "1.0.1",
    "@fortawesome/fontawesome-free": "^5.2.0",
    "@types/d3-color": "1.0.5",
    "@types/jasmine": "2.5.54",
    "@types/jasminewd2": "2.0.3",
    "@types/leaflet": "1.2.3",
    "@types/node": "^12.11.1",
    "codelyzer": "^6.0.0",
    "conventional-changelog-cli": "1.3.4",
    "husky": "0.13.3",
    "jasmine-core": "~3.5.0",
    "jasmine-spec-reporter": "~5.0.0",
    "karma": "~5.0.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-cli": "1.0.1",
    "karma-coverage-istanbul-reporter": "~3.0.2",
    "karma-jasmine": "~3.3.0",
    "karma-jasmine-html-reporter": "^1.5.0",
    "npm-run-all": "4.0.2",
    "protractor": "~7.0.0",
    "rimraf": "2.6.1",
    "stylelint": "7.13.0",
    "ts-node": "3.2.2",
    "tslint": "~6.1.0",
    "tslint-language-service": "^0.9.9",
    "typescript": "3.9.7"
  }
}

```

**Comment by Faisal49m** [comment]
Same thing here

**Comment by georgetrad** [comment]
@Tavethiya @Faisal49m 

Did you try updating nebular? the datetime-picker wasn't available in nebular 6.0.0. I updated to 7.0.0 and everything is working fine.

Here's the [release notes](https://github.com/akveo/nebular/blob/master/CHANGELOG.md).
